pages/index.html

```diff
@@ -4,12 +4,41 @@
 
 {% block title %}{% trans "Free DJ Mixing Software App" %}{% endblock %}
 
+{% block extra_script %}
+<script type="text/javascript" src="{% static '/static/js/gallery.js' %}"></script>
+<script type="text/javascript">
+  const images = [
+    "{% static '/static/images/feature_skins_deere.png' %}",
+    "{% static '/static/images/feature_skins_latenight.png' %}",
+    "{% static '/static/images/feature_skins_shade.png' %}",
+    "{% static '/static/images/feature_skins_tango.png' %}"
+  ];
```

Comment:
Can this be avoided? Or is hardcoding this not reliable? I believe urls like these are already hardcoded in the css, so I would consider moving this initialization code into `gallery.js`

Reply:
I wanted to keep `gallery.js` as general as possible to be reused elsewhere. That's why those URLs are not in `gallery.js`. I can put them in a separate JS file. But those files might only be temporary ones for the first test.
The original image has some kind of "border darkness" effect, which we might want to have again on the frontpage. 

At the location where those images are used in the first place (features.html) they are hardcoded in the HTML. Should the URLS be moved to a separate JS?